### PR TITLE
Add control plane service implementation

### PR DESCRIPTION
**Pull request by HindujaB**
## Purpose
$subject

## Automation tests
- Unit tests
  > Code coverage information
- Integration tests
  > Details about the test cases and coverage

## Security checks
- Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
- Ran FindSecurityBugs plugin and verified report? yes/no
- Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: native/src/main/java/io/ballerina/lib/wso2/controlplane/Utils.java [hunk 1]
```diff
@@ -16,21 +16,150 @@
...
+    private static Object getServiceAnnotations(BObject serviceObj) {
+        BMap<BString, Object> mapValue = getMapAnydataValue();
+        AnnotatableType serviceType = (AnnotatableType) TypeUtils.getImpliedType(serviceObj.getOriginalType());
+        for (Map.Entry<BString, Object> entry : serviceType.getAnnotations().entrySet()) {
+            mapValue.put(entry.getKey(), entry.getValue());
+        }
+        return mapValue;
+    }
+
+
```
Comment (gabilang) [inline native/src/main/java/io/ballerina/lib/wso2/controlplane/Utils.java]:
```suggestion
```

File: ballerina-tests/build.gradle [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+    if (project.hasProperty('balTests')) {
+        testPackages = project.findProperty('balTests').toString().split(",")
+    }
+    if (project.hasProperty('skipBalTests')) {
+        project.findProperty('skipBalTests').toString().split(",").each {testPackage ->
+            testPackages.remove(testPackage)
+        }
+    }
+
+
```
Comment (gabilang) [inline ballerina-tests/build.gradle]:
```suggestion
```

**Comment by gabilang** [comment]
Since there are no usages for compiler plugin here, we can remove the already added package for compiler-plugin.

File: ballerina-tests/artifacts-tests/tests/test.bal [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    cp:Artifacts|error artifacts = rmClient->get("management/Service");
+    test:assertTrue(artifacts is cp:Artifacts, "Invalid response received");
+    test:assertTrue(artifacts.count() == 1, "No services found");
+}
+
+service /hello on new http:Listener(testPort) {
+    resource function get greeting() returns string {
+        return "Hello, World!";
+    }
+
```
Comment (SasinduDilshara) [inline ballerina-tests/artifacts-tests/tests/test.bal]:
```suggestion
```

**Review by gabilang — APPROVED** [review]

**Pull request merged**